src/components/activity/Navbar.js

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import {Link, useMatch, useResolvedPath} from 'react-router-dom';
+
+function Navbar() {
+  return (
+    <StyledNavigation>
+      <>
+        <main>
```

Comment:
You have put a `main` element inside a `main` element ("StyledNavigation", which is not a good name because it's not a navigation).